models/articles.model.js

```diff
@@ -42,3 +42,26 @@ exports.fetchArticles = async (article_id) => {
   });
   return articles.rows;
 };
+
+exports.articlePatcher = async (article_id, inc_votes) => {
+  const oldVotes = await db.query(
+    `
+  SELECT votes FROM articles WHERE article_id = $1
+  `,
+    [article_id]
+  );
+  const insertVotes = await db.query(
+    `
+  UPDATE articles SET votes = $1
```

Comment:
This all works fine, but if you were to replace the `=` with a `+`, it would mean you don't have to do the first SQL query of `oldVotes`! 😄